src/components/app-pagination/AppPagination.tsx

```diff
@@ -1,17 +1,29 @@
+import { ChangeEvent, FC } from 'react'
+
 import Pagination from '@mui/material/Pagination'
 import Box from '@mui/system/Box'
+
+import { Size } from '~/types'
 import { styles } from '~/components/app-pagination/AppPagination.styles'
 
-const AppPagination = ({
-  size = 'medium',
+interface AppPaginationProps {
+  size?: Size
+  page: number
+  itemsCount: number
+  pageSize: number
+  setCurrentPage: (page: number) => void
+}
+
+const AppPagination: FC<AppPaginationProps> = ({
+  size = Size.Medium,
   page,
   itemsCount,
   pageSize,
```

Comment:
also pageSize is not cleat, itemsPerPage will be better

Reply:
changed